runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/StreamingDataflowWorker.java

```diff
@@ -785,6 +779,18 @@ private int chooseMaximumNumberOfThreads() {
     return MAX_PROCESSING_THREADS;
   }
 
+  private int chooseMaximumBundlesOutstanding() {
+    return Math.max(options.getMaxBundlesFromWindmillOutstanding(), chooseMaximumNumberOfThreads());
```

Comment:
I think the previous was 100 + chooseMaximumNumberOfThreads(), should we keep that?
The requested was fixed 400 previously but the active+queued previously was 100 queued+num threads active.

Reply:
Previous was 100 + MAX_PROCESSING_THREADS (the constant 300) and didn't take into account the number of actual processing threads.

This keeps the identical behavior.

Reply:
I've fixed this to match the current behavior in the case that the number of threads is modified.